clients/src/main/java/org/apache/kafka/clients/consumer/WakeupException.java

```diff
@@ -14,7 +14,7 @@
 
 import org.apache.kafka.common.KafkaException;
 
-public class ConsumerWakeupException extends KafkaException {
+public class WakeupException extends KafkaException {
```

Comment:
Happy to. The only reason I kept it here was that it was only used in ConsumerNetworkClient.